misk/src/main/kotlin/misk/config/ConfigWebModule.kt

```diff
@@ -2,9 +2,23 @@ package misk.config
 
 import misk.inject.KAbstractModule
 import misk.web.WebActionModule
+import misk.web.actions.AdminTab
+import misk.web.actions.UpstreamResourceInterceptor
+import okhttp3.HttpUrl
 
 class ConfigWebModule : KAbstractModule() {
   override fun configure() {
     install(WebActionModule.create<ConfigAdminAction>())
+    multibind<AdminTab>().toInstance(AdminTab(
+        "Config",
+        "config",
+        "/_admin/config/"
+    ))
+    multibind<UpstreamResourceInterceptor.Mapping>().toInstance(UpstreamResourceInterceptor.Mapping(
```

Comment:
TODO: we only wanna do this in development

Reply:
Good point, would an environment variable check on production/dev mode be the way to do that?

Reply:
That’s probably the most capable way to do it. Lets leave as-is for now and follow-up.